compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/infer-effect-dependencies/bailout-retry/error.callsite-in-non-react-fn.expect.md

```diff
@@ -0,0 +1,26 @@
+
+## Input
+
+```javascript
+// @inferEffectDependencies @compilationMode(infer) @panicThreshold(none)
```

Comment:
Note that these fixtures error even with `@panicThreshold(none)`. This PR changes `inferEffectDeps` to be a "required" feature (e.g. if enabled, relevant code must either be transformed or fail compilation)